### PR TITLE
i18n(zh-cn): translate decap-cms.md

### PR DESCRIPTION
**Pull request by HashCookie**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate decap-cms.md

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/decap-cms.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+type: cms
+stub: true
+service: Decap CMS
+i18nReady: true
+---
+
+[Decap CMS](https://www.decapcms.org/)（前身为 Netlify CMS）是一个基于 Git 的开源内容管理系统。
+## 社区资源 
+
+- 使用[`astro-netlify-cms`](https://github.com/delucis/astro-netlify-cms) Astro 集成，将 Netlify CMS 添加到你的项目中。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/decap-cms.mdx:13]:
```suggestion
- 使用 [`astro-netlify-cms`](https://github.com/delucis/astro-netlify-cms) Astro 集成，将 Netlify CMS 添加到你的项目中。
```

File: src/content/docs/zh-cn/guides/cms/decap-cms.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- 你可以从[Astro 博客入门与 Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter)开始。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/decap-cms.mdx:15]:
```suggestion
- 你可以从 [Astro 博客入门与 Netlify CMS](https://github.com/delucis/astro-netlify-cms-starter) 开始。
```

File: src/content/docs/zh-cn/guides/cms/decap-cms.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+- 博客文章：[使用基于 Git 的 CMS 管理 Astro 站点内容](https://aalam.vercel.app/blog/astro-and-git-cms-netlify) 作者：Aftab Alam
+
+- YouTube教程：[在几分钟内使用 Astro 和 Netlify CMS创建自定义博客！](https://www.youtube.com/watch?v=3yip2wSRX_4) 作者：Kumail Pirzada
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/cms/decap-cms.mdx:19]:
```suggestion
- YouTube教程：[在几分钟内使用 Astro 和 Netlify CMS 创建自定义博客！](https://www.youtube.com/watch?v=3yip2wSRX_4) 作者：Kumail Pirzada
```

**Review by 100gle — CHANGES_REQUESTED** [review]
需要简单调整一下部分排版

**Review by 100gle — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 💜

**Pull request merged**